synapse/media/preview_html.py

```diff
@@ -247,7 +255,7 @@ def parse_html_to_open_graph(tree: "etree.Element") -> Dict[str, Optional[str]]:
     # "og:video:height" : "720",
     # "og:video:secure_url": "https://www.youtube.com/v/LXDBoHyjmtw?version=3",
 
-    og = _get_meta_tags(tree, "property", "og")
+    og: Dict[str, Optional[str]] = _get_meta_tags(tree, "property", "og")
```

Comment:
I guess we need to explicitly annotate this because it is a recursive self-call?

Reply:
I actually don't think we need this anymore based on backing out other changes.